sdk/src/main/java/com/deploygate/service/DeployGateEvent.java

```diff
@@ -31,6 +31,9 @@
     public static final String EXTRA_LOG = "log";
     public static final String EXTRA_LOG_TYPE = "logType";
     public static final String EXTRA_EXCEPTION = "exception";
```

Comment:
should I add `Deprecated`?

Reply:
Addressed in https://github.com/DeployGate/deploygate-android-sdk/pull/14/commits/9ea3e03a209116d098c035cdfb49d40ae9d481da